PrettyPrint/markdown_file.md

```diff
@@ -0,0 +1,126 @@
+
+ARM 01, 001
+---
+###### metadata
+    Primary publication: ARM 01, 001
+ 	Author(s): Dossin, Georges
+ 	Publication date: 1946
+ 	Secondary publication(s): Durand, Jean-Marie, LAPO 16, 0305
+ 	Collection: National Museum of Syria, Damascus, Syria
+ 	Museum no.: NMSD —
+ 	Accession no.:
+ 	Provenience: Mari (mod. Tell Hariri)
+ 	Excavation no.:
+ 	Period: Old Babylonian (ca. 1900-1600 BC)
+ 	Dates referenced:
+ 	Object type: tablet
+ 	Remarks:
+ 	Material: clay
+ 	Language: Akkadian
+ 	Genre: Letter
+ 	Sub-genre:
+ 	CDLI comments:
+ 	Catalogue source: 20050104 cdliadmin
+ 	ATF source: cdlistaff
+ 	Translation: Durand, Jean-Marie (fr); Guerra, Dylan M. (en)
+ 	UCLA Library ARK: 21198/zz001rsp8x
+ 	Composite no.:
+ 	Seal no.:
+ 	CDLI no.: P254202
+###### transliteration
+    &P254202 = ARM 01, 001
+ 	#atf: lang akk
+ 	@tablet
+ 	@obverse
+ 	1. a-na ia-ah-du-li-[im]
```

Comment:
Should these render as an ordered list? They currently are not.

Reply:
When I try for this (`transliteration = '\n> '.join(text['transliteration'][0])`), anytime a non number shows up (ln 43, `$ rest broken`), the code breaks. 

Advice? 

![image](https://user-images.githubusercontent.com/38022737/42959036-3fa5eb64-8b3c-11e8-901b-7ec711ab4b1a.png)

Reply:
Looks like you need to clean the text up and remove the `'` before each number.

Reply:
The apostrophe denotes the back side of a tablet and is standard convention for publication. In this case, would placing a space between number and apostrophe be a valid solution?

Reply:
No because markdown list items are in the form `1.`. You can just bold the line number with the apostrophe if you want to preserve it while formatting as though it is an ordered list.